python/tk_desktop/desktop_window.py

```diff
@@ -504,9 +503,11 @@ def show_update_project_config(self):
         self.update_project_config_widget.show()
         self.project_overlay.hide()
 
+    def _get_current_user(self):
+        return login.get_current_user(sgtk.platform.current_engine().tank)
```

Comment:
This code is part of the engine, but I don't understand what's the difference between calling the two? current_engine has the merit of being explicit about what I think I am getting back.